docs/results.md

```diff
@@ -0,0 +1,376 @@
+
+The results for Scale test runs can be found in the [`results/scale`](https://github.com/puppetlabs/gatling-puppet-load-test/tree/master/results/scale) directory.
+The results for each Scale test run are contained within a separate timestamped sub-directory of the `results/scale` directory.
+The name of each result directory consists of the label `SCALE_` to indicate a Scale run followed by a timestamp generated at the start of the run.
+The directory also contains a `latest` link pointing to the latest Scale test results.
+
+```
+results/scale
+├── PERF_SCALE_1572446292
+└── latest -> /home/centos/gplt/slv-680/gatling-puppet-load-test/results/scale/PERF_SCALE_1572446292
+
+```
+
+Each Scale test result directory contains directories and files related to the overall Scale test run as well as each iteration of the scenario.
+For example:
+```
+results/scale/PERF_SCALE_1572446292
+├── PERF_SCALE_1572446292.csv
+├── PERF_SCALE_1572446292.csv.html
+├── Scale_1572446292_1_600
+├── Scale_1572446292_2_700
+├── Scale_1572446292_3_800
+├── beaker_environment.txt
+├── json
+├── log
+├── pe_tune_current.txt
+└── puppet-metrics-collector
+```
+
+## Scale test report
+The Scale test results contains a CSV file with measurements extracted from the Gatling JSON data for each iteration.
+This report is useful for comparing the performance impact of additional nodes for each iteration.
+It is also useful for comparing the outcome of multiple Scale test runs.
+
+In the example above this file is `PERF_SCALE_1572446292.csv`; 
+as with other CSV files in the results it has been converted to an HTML document (`PERF_SCALE_1572446292.csv.html`).
+
+## Beaker environment settings
+The `beaker_environment.txt` file contains the environment variables for the test run.
+
+## Iterations
+Each iteration of a Scale test run is actually a separate Performance test run and will create a separate result in the `results/perf` directory.
+The Scale results are then constructed by selectively copying specific directories and files from the results for each iteration of the run in order to provide a cohesive overview of the entire Scale test run and to overcome the somewhat inconvenient organizational structure of the Performance results.
+
+A separate directory is created within the Scale test results for each iteration, labeled with the following pattern:
+```
+Scale_{timestamp}_{iteration}_{number_of_agents}
+```
+
+This pattern makes it easy to tell how many iterations were run and how many agents were run per iteration.
+For example, the directory listing above is from a sample test with three iterations:
+```
+results/scale/PERF_SCALE_1572446292
+├── Scale_1572446292_1_600
+├── Scale_1572446292_2_700
+└── Scale_1572446292_3_800
+```
+
+### Iteration sub-directories
+The sub-directory created for each Scale test iteration contains directories and files selectively copied from the corresponding Performance results for that iteration.
+For example:
```

Comment:
I think you want a blank line before this so that it doesn't get folded into the previous paragraph when rendered.

Reply:
Added a blank line before each 'For example'.